src/components/CharacterCard.js

```diff
@@ -1,5 +1,27 @@
 import React from "react";
+import {
+   Col,
+    Card,
+    CardImg,
+    CardText,
+    CardBody, 
+    CardTitle
+} from "reactstrap";
 
-export default function CharacterCard() {
-  return <span>todo: character</span>;
+const CharacterCard = (props) => {
+  return (
+    <div>
+      <Col>
+     <Card className="character-card" outline color="warning">
+        <CardImg top width="25%" src={props.image} alt="character" />
+        <CardBody>
+        <CardTitle>Name: {props.name}</CardTitle>
+        <CardText>Species: {props.species}</CardText>
+      </CardBody>
+     </Card>
+     </Col>
+    </div>
+  );
 }
```

Comment:
Proper indentation:
```
const CharacterCard = props => {
  return (
    <div>
      <Col>
        <Card className="character-card" outline color="warning">
          <CardImg top width="25%" src={props.image} alt="character" />
          <CardBody>
            <CardTitle>Name: {props.name}</CardTitle>
            <CardText>Species: {props.species}</CardText>
          </CardBody>
        </Card>
      </Col>
    </div>
  );
};
```
**Tip:** _Opening tags and closing tags `<div>` `</div>` are always on the same indentation level_